cookbooks/aws-parallelcluster-config/recipes/compute_scheduler.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: compute_slurm
+#
+# Copyright 2013-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Mount /opt/parallelcluster/shared over NFS
+mount node['cluster']['scheduler']['opt_shared_path'] do
```

Comment:
This could be done on `aws-parallelcluster-config/recipes/prep_env_compute_byos.rb`defined in https://github.com/aws/aws-parallelcluster-cookbook/pull/1135